server/app.js

```diff
@@ -14,8 +14,18 @@ app.get('/:id', (req, res) => {
   res.render('../public/index.html');
 });
 
-app.post('/reviews/:expId', (req, res) => {
-  db.createExpReviews(req.params.expId, req.body, (err, reviewId) => {
+app.get('/reviews/:id', (req, res) => {
+  db.getExpReviews(req.params.id, (err, reviews) => {
+    if (err) {
+      res.status(400).json(err);
+    } else {
+      res.status(200).json(reviews);
+    }
+  });
+});
+
+app.post('/review/', (req, res) => {
+  db.createExpReview(req.body, (err, reviewId) => {
     if (err) {
       res.status(400).json(err);
     } else {
```

Comment:
What happens if there is no error?

Reply:
Maybe status 200?

Reply:
If post is good it will send back the new review id

Reply:
Sorry I hadn't seen line 32.